File: appjava/src/main/java/com/byoutline/kickmaterial/model/ProjectPhoto.java
package com.byoutline.kickmaterial.model;

import com.google.gson.annotations.SerializedName;
import org.parceler.Parcel;

/**
 * Created by Sebastian Kacprzak on 25.03.15.
 */
@Parcel(Parcel.Serialization.FIELD)
public class ProjectPhoto {
    public String full;
    public String ed;
    public String med;
    public String little;
    public String small;
    public String thumb;
    @SerializedName("1024x768")
    public String size1024x768;
    @SerializedName("1536x1152")
    public String size1536x1152;
}

File: app/src/main/java/com/byoutline/kickmaterial/model/ProjectPhoto.kt
package com.byoutline.kickmaterial.model

import com.google.gson.annotations.SerializedName
import paperparcel.PaperParcel
import paperparcel.PaperParcelable

/**
 * Created by Sebastian Kacprzak on 25.03.15.
 */
@PaperParcel
class ProjectPhoto: PaperParcelable  {
    var full: String? = null
    var ed: String? = null
    var med: String? = null
    var little: String? = null
    var small: String? = null
    var thumb: String? = null
    @SerializedName("1024x768")
    var size1024x768: String? = null
    @SerializedName("1536x1152")
    var size1536x1152: String? = null

    companion object {
        @JvmField val CREATOR = PaperParcelProjectPhoto.CREATOR
    }
}
